tutorials/imad-tutorial-pt1/index.ipynb

```diff
@@ -0,0 +1,2078 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    aoi = ee.Geometry(aois.geometries().get(26))
</pre><p> </p><p>This is sort of a bad convention in GEE, is their a property you can filter on e.g., <code>aois.filter(...).geometry()</code>, instead of using <code>aois.geometries.get(index)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/583/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Listing the properties of aois all I get is ['system:version', 'system:id', 'system:asset_size']. Left as is

Reply:
Acknowledged